commands/update_builder.go

```diff
@@ -99,12 +99,20 @@ func updateBuilderRun(flags updateBuilderFlags) error {
 
 	builder.Lifecycle.Version = lifecycleImage.Version
 
-	buildImage, err := internal.FindLatestBuildImage(builder.Stack.RunImage, builder.Stack.BuildImage)
+	runImage, buildImage, err := internal.FindLatestStackImages(builder.Stack.RunImage, builder.Stack.BuildImage)
 	if err != nil {
 		return err
 	}
 
 	builder.Stack.BuildImage = fmt.Sprintf("%s:%s", buildImage.Name, buildImage.Version)
+	if runImage != (internal.Image{}) {
```

Comment:
Are the `()` necessary?

Reply:
I think you need them when comparing an instantiation inline but that could be old news.

Reply:
yes they are necessary unfortunately. `expected ';', found '{' (typecheck)`